teamscale-client/src/main/java/com/teamscale/client/PrioritizableTest.java

```diff
@@ -13,6 +15,18 @@
 	/** The reason the test has been selected. */
 	public String selectionReason;
 
+	/** Partition of the test. */
+	public String partition;
+
+	/** Duration in ms. May be null if not set. */
+	public Long durationInMs;
+
+	/**
+	 * The score determined by the TIA algorithm.
```

Comment:
Please comment the semantics that we can guarantee. I think at least we should mention if higher = better score or lower = better score. Maybe there's more? Like the range 0-1 or 0-Infinity or something?